.github/workflows/ci.yml

```diff
@@ -160,14 +160,11 @@ jobs:
 
       - name: Try extracting i18n strings (Python)
         if: success() || failure()
-        run: indico i18n extract-messages
+        run: indico i18n extract indico --python
 
       - name: Try extracting i18n strings (JS)
         if: success() || failure()
-        run: |
-          echo "::add-matcher::.github/matchers/react-jsx-i18n-problem-matcher.json"
```

Comment:
The matcher stuff needs to stay. It's used to show nice annotations in the PR view when there's an error from the extraction.

Reply:
My bad, reverting that change.